### DISCUSSION TITLE
Built-in speaker diarization?

### DISCUSSION THREAD

**Discussion by martinmueller4voice** [openai/whisper#1552] [General]
Hi community!
I had whisper (most recent version from GitHub, commit b91c907694f96a3fb9da03d4bbdc83fbcd3a40a4) transcribe a German test recording using this call:
`!whisper --language de --model large --suppress_tokens 0,11,13,30 audio.wav`
(--suppress_tokens to suppress output of auto-punctuation) and was surprised to see the transcript beginning with:
`(Rednerwechsel) Sehr geehrte Damen`
("Rednerwechsel" means change of speaker)
This change of speaker seems to be token 50364 (if I interprete the json file correctly), if this is of any help.

So far, I've always transcribed single speaker recordings and the one above is no exception.
In addition, this (Rednerwechsel) appears at the very beginning of the transcription, so there was no other speaker talking before.
Other recordings from the same speaker don't show this output.

I always thought Whisper doesn't have speaker diarization out of the box, so where does this token come from?

**Comment by phineas-pta** [comment]
likely training data problem like #928

**Reply by martinmueller4voice** [reply]
I just tried with large-v1 and there the "(Rednerwechsel)" did not happen.
